model/latest_entry.go

```diff
@@ -55,3 +55,7 @@ func (l *LatestEntry) Get(ctx context.Context) error {
 	g := goon.FromContext(ctx)
 	return g.Get(l)
 }
+
+func ReviewdogTest() error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ReviewdogTest should have comment or be unexported